src/main/resources/META-INF/plugin.xml

```diff
@@ -17,6 +17,7 @@
     </change-notes>
 
     <depends>com.intellij.modules.java</depends>
+    <depends>org.intellij.scala</depends>
```

Comment:
exposing strict dependency on scala plugin (by ID), now our plugin won't even start wo Scala plugin installed/enabled

Reply:
This is the only part I don't like about this approach... But having a working REPL is way higher in the priority list than making the plugin usable for non-scala-courses so I'm completely fine with using this approach for now